server/app/models/OrderModel.ts

```diff
@@ -0,0 +1,22 @@
+import { Document, Schema } from 'mongoose';
+
+export interface IOrder extends Document {
+  componentName: string;
+  option1: string;
+  option2: string;
+  quantity: number;
+  cost: number;
+  deliveryDate: Date;
+  manufactureName: string;
+  note: string;
+}
```

Comment:
Should we store `orderDate` as well?

Wasn't the ordering page about ordering materials? And the production page was about parts and bikes.

Reply:
uhhh idk i based it off the ligma. but yessir order date should be on there.